docs/variants/novacustom_ns5x_tgl/releases.md

```diff
@@ -13,6 +13,85 @@ For details about our release process please read
 
 </center>
 
+## v1.5.2 - 2024-01-16
+
+Test results for this release can be found
+[here](https://docs.google.com/spreadsheets/d/1LOXY9HCu-fMitkYwX08iLsQdSNenzyU0LnMdVbZB5Do/edit#gid=38447675).
+
+### Added
+
+- [Added Microsoft 2023 keys for UEFI Secure Boot and DBX from 2023.05.9](https://github.com/Dasharo/edk2/commit/b7274c98697e972e772236caf830c0780ec498bd)
+
+### Changed
+
+- [Changed throttling temperature to 80 degrees](https://github.com/Dasharo/dasharo-issues/issues/596)
+- [Enabled HTTPS for Dasharo Tools Suite network boot option](https://github.com/Dasharo/dasharo-issues/issues/450)
+
+### Fixed
+
+- [Low performance when booting on battery power only](https://github.com/Dasharo/dasharo-issues/issues/555)
+- [Regression charging via dell usb-c dock](https://github.com/Dasharo/dasharo-issues/issues/587)
+- [Video on USB-C docks sometimes fails to initialize](https://github.com/Dasharo/dasharo-issues/issues/560)
+- [USB-C chargers may be overloaded in certain scenarios](https://github.com/Dasharo/dasharo-issues/issues/619)
+- [Hybrid Power Boost mode of battery charger is not functional](https://github.com/Dasharo/dasharo-issues/issues/620)
+- [Laptop may shut down when disconnecting power supply](https://github.com/Dasharo/dasharo-issues/issues/621)
+- [Input current limits are set too high](https://github.com/Dasharo/dasharo-issues/issues/622)
+- [Keyboard backlight does not turn off immediately when going to sleep](https://github.com/Dasharo/dasharo-issues/issues/623)
+- [Power LED blinks erratically in HPB mode](https://github.com/Dasharo/dasharo-issues/issues/624)
+- [EC applying power limits may sometimes fail](https://github.com/Dasharo/dasharo-issues/issues/625)
+- [Wrong power limits are applied on AC attach / detach](https://github.com/Dasharo/dasharo-issues/issues/626)
+- [USB-PD controller may hang in glitched state on shutdown](https://github.com/Dasharo/dasharo-issues/issues/627)
+- [Yellow bangs in device manager](https://docs.dasharo.com/unified/clevo/post-install/#unrecognized-usb-controller-device-in-device-manager)
+- [Random freezes in Windows 11](https://github.com/Dasharo/dasharo-issues/issues/628)
+- [Low level interfering crackling/popping of the speakers while playing no sound](https://github.com/Dasharo/dasharo-issues/issues/224)
+- [ACPI error during boot (system76_acpi::kbd_backlight)](https://github.com/Dasharo/dasharo-issues/issues/559)
+- [Thunderbolt may not work after wakeup from sleep](https://github.com/Dasharo/dasharo-issues/issues/629)
+- [Unnecessary security warning after firmware update](https://github.com/Dasharo/dasharo-issues/issues/556)
+- [Early boot DMA protection setting in UEFI does not work correctly](https://github.com/Dasharo/dasharo-issues/issues/630)
```

Comment:
@mkopec fixed or know issue? https://github.com/Dasharo/dasharo-issues/issues/548#issuecomment-1878785880

Reply:
@BeataZdunczyk right, moved to known issues for ns5x and nv4x tgl: https://github.com/Dasharo/docs/pull/739/commits/2010062bfa8cac881e3fc0bdf65f53c71e4f9ae0